patches/0009-Fix-794-with-an-unconditional-read.patch

```diff
@@ -5,10 +5,10 @@ Subject: [PATCH] Fix #794 with an unconditional read.
 
 
 diff --git a/google-beta/services/sql/resource_sql_database_instance.go b/google-beta/services/sql/resource_sql_database_instance.go
-index 4ee791de5..1ee2d9ec6 100644
+index da372674a..db76a3d3e 100644
```

Comment:
Do we need all these changes to patches?

Reply:
That is what the current process generates. They do provide more information.

Reply:
Seems like this only pollutes the history - what additional information is provided here?

Reply:
The last commit that the patch cleanly applied to.

Reply:
That seems unlikely to be useful given that all patches must have applied cleanly to the previously released version.

I'd say the git history is more useful here - can we please revert the changes here?

Reply:
As I said, this is what the current process generates. Running `./upstream.sh checkout && ./upstream.sh check_in` will restore the changes. I would rather add to the git history here then have it added in a future release, since the change is caused at this release. It will be added at the next automatic release regardless if we manually remove it here.

I don't love the extra noise, but this isn't the place to fix it. That would be `ci-mgmt`.